src/lib/API/api.js

```diff
@@ -117,6 +117,20 @@ class API {
     return this.client.post('/verify/user', { verificationData })
   }
 
+  /**
+   * `ip-api.com/json` get location api call
+   */
+  getLocation(): AxiosPromise<any> {
+    return axios
+      .get('https://get.geojs.io/v1/ip/country.json')
+      .then(response => response)
+      .catch(error => {
+        // Do something with response error
+        log.error('axios response error', { error })
+        return Promise.reject(error)
+      })
```

Comment:
This `then.catch` is unnecessary as the error is already captured on the function call.

Catching the error in the function call is enough, and we'll keep a code consistency as this is the only method in the API library that's catching the error.